content/en/apps/examples/contact-tracing.md

```diff
@@ -26,18 +26,20 @@ Contact Tracing with the CHT offers a rapidly scalable surveillance strategy by
 * Referring symptomatic cases for skilled care following triage
 * Engaging in community education and follow-up during isolation
 
-## App User Roles Example
+## Users and Hierarchy Example
 
-![user-roles.png](user-roles.png)
+| User                  | Location             | Devices                             | Role                                                                                                                                                                                                                                              |
+| :-------------------- | :------------------- | :---------------------------------- | :------------------------------------------------------------------------------------------------- |
+| Data Entry  | Facility level       | Desktop, laptop or tablet           | Responsible for ensuring all contacts are traced. Create contact list based on confirmed cases. Assign contacts to Tracers for follow-up. Notified of unsuccessful traces and symptomatic cases. |
+| Tracer                | Community level      | Smartphone                          | Responsible for tracking contacts of confirmed case. View list of assigned contacts registered with Index Patient Screen contacts and submit Trace follow-up forms/tasks. |
```

Comment:
```suggestion
| Tracer                | Community level      | Smartphone                          | Responsible for tracking contacts of confirmed case. View list of assigned contacts registered with Index Patient Screen contacts and submit Tracer follow-up forms/tasks. |
```

Reply:
I'm not familiar with "Index Patient Screen" and it's not defined in this example. It may be worth clarifying if you think other users would also not be familiar

Reply:
@MaxDiz updated the copy pulling from the summary deck, LMK if this is clearer